console/ui/src/app/config/config.component.ts

```diff
@@ -131,6 +131,53 @@ export class ConfigComponent implements OnInit, OnDestroy {
     }
   }
 
+  // If the file is in JSON, it stringifies the value of the storage object.
+  private processFile(fIn: File, callback: (f: File) => void): void {
```

Comment:
Whilst this is really good, I'd suggest that we make this much simpler - i.e. make the server code such that it only accepts `value` as a JSON object. This is indeed a breaking change to the behaviour of this API from Nakama2 but I think it's worth it to make it much much easier to deal with the data.

Reply:
I've made the change server side instead.